README.md

```diff
@@ -116,7 +116,7 @@ func Helloworld(g *gin.Context)  {
 
 func main()  {
    r := gin.Default()
-   docs.SwaggerInfo.BasePath = "/api/v1"
+   docs.SwaggerInfo_swagger.BasePath = "/api/v1"
```

Comment:
I mistakenly generated the doc with 1.7.9 instead of 1.7.9p1 which has undone this change for default instance in order to prevent confusion.

Would you be kind to generate the docs with the latest version of swag and update the PR?